Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake

```diff
@@ -1,7 +1,28 @@
+function(expand_list_with_globbing list_name)
+  set(input_list ${${list_name}})
+#  message(STATUS "expand_list_with_globbing(${list_name}), ${list_name} is: ${input_list}")
+  list(LENGTH input_list list_lenght)
+  math(EXPR list_last_n "${list_lenght} - 1")
+  set(output_list)
+  foreach(n RANGE ${list_last_n})
+#    message(STATUS "n=${n}")
+    list(GET input_list ${n} item_n)
+#    message(STATUS "argument ${n} is ${item_n}")
+    if(item_n MATCHES ".*\\*.*")
+      file(GLOB files RELATIVE ${CGAL_CURRENT_SOURCE_DIR} ${item_n})
+      list(APPEND output_list ${files})
+    else()
+      list(APPEND output_list ${item_n})
+    endif()
+#    message(STATUS "  new value of the output list: ${output_list}")
+  endforeach()
+  set(${list_name} ${output_list} PARENT_SCOPE)
+endfunction()
+
```

Comment:
Done in https://github.com/CGAL/cgal/pull/1357/commits/bd156440a090700171826c9b0787a16f91506a79.